src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -28,19 +28,19 @@
  *
  * @package Facebook
  */
-
 class GraphAlbum extends GraphNode
 {
     /**
-     * @var array Maps object key names to Graph object types.
+     * @var array Maps object key names to GraphNode types.
      */
     protected static $graphObjectMap = [
-        'from' => '\Facebook\GraphNodes\GraphUser',
```

Comment:
mmmm, so we need to return the `form` auto cast and add todo comment to delete it later in a the next major release?